<?xml version="1.0" encoding="UTF-8"?><testrun name="Rerun org.magnum.dataup.AutoGradingTest.testAddVideoMetadata" project="Asgn1-VideoUp" tests="1" started="1" failures="0" errors="1" ignored="0">
  <testcase name="testAddVideoMetadata" classname="org.magnum.dataup.AutoGradingTest" time="1.027">
    <error>retrofit.RetrofitError: Connection refused: connect&#13;
	at retrofit.RetrofitError.networkError(RetrofitError.java:27)&#13;
	at retrofit.RestAdapter$RestHandler.invokeRequest(RestAdapter.java:389)&#13;
	at retrofit.RestAdapter$RestHandler.invoke(RestAdapter.java:240)&#13;
	at com.sun.proxy.$Proxy9.addVideo(Unknown Source)&#13;
	at org.magnum.dataup.AutoGradingTest.testAddVideoMetadata(AutoGradingTest.java:103)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)&#13;
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)&#13;
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
Caused by: java.net.ConnectException: Connection refused: connect&#13;
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)&#13;
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)&#13;
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)&#13;
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)&#13;
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)&#13;
	at java.net.PlainSocketImpl.connect(Unknown Source)&#13;
	at java.net.SocksSocketImpl.connect(Unknown Source)&#13;
	at java.net.Socket.connect(Unknown Source)&#13;
	at sun.net.NetworkClient.doConnect(Unknown Source)&#13;
	at sun.net.www.http.HttpClient.openServer(Unknown Source)&#13;
	at sun.net.www.http.HttpClient.openServer(Unknown Source)&#13;
	at sun.net.www.http.HttpClient.&lt;init&gt;(Unknown Source)&#13;
	at sun.net.www.http.HttpClient.New(Unknown Source)&#13;
	at sun.net.www.http.HttpClient.New(Unknown Source)&#13;
	at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(Unknown Source)&#13;
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(Unknown Source)&#13;
	at sun.net.www.protocol.http.HttpURLConnection.connect(Unknown Source)&#13;
	at sun.net.www.protocol.http.HttpURLConnection.getOutputStream(Unknown Source)&#13;
	at retrofit.client.UrlConnectionClient.prepareRequest(UrlConnectionClient.java:68)&#13;
	at retrofit.client.UrlConnectionClient.execute(UrlConnectionClient.java:37)&#13;
	at retrofit.RestAdapter$RestHandler.invokeRequest(RestAdapter.java:321)&#13;
	... 26 more&#13;
&#13;
</error>
  </testcase>
</testrun>
